ee/agent/storage/sqlite/keyvalue_store_sqlite.go

```diff
@@ -19,20 +19,23 @@ import (
 	_ "modernc.org/sqlite"
 )
 
-type storeName int
+type StoreName int
 
 const (
-	StartupSettingsStore storeName = iota
+	StartupSettingsStore StoreName = iota
+	WatchdogLogStore     StoreName = 1
 )
 
 var missingMigrationErrFormat = regexp.MustCompile(`no migration found for version \d+`)
 
 // String translates the exported int constant to the actual name of the
 // supported table in the sqlite database.
-func (s storeName) String() string {
+func (s StoreName) String() string {
 	switch s {
 	case StartupSettingsStore:
 		return "startup_settings"
+	case WatchdogLogStore:
+		return "watchdog_logs"
```

Comment:
If we want to push other logs here (say early startup logs) how would we do it? Would we make a new store? Accept it as slightly misnamed?